main.go

```diff
@@ -315,7 +317,21 @@ func main() {
 			log.Printf("%v\n", err)
 			os.Exit(64)
 		}
-		exitCode := RunWorker()
+		monitor = NewMonitor(
+			config.Project,
+			tcauth.New(&tcclient.Credentials{
+				ClientID:    config.ClientID,
+				AccessToken: config.AccessToken,
+				Certificate: config.Certificate,
+			}),
+			config.LogLevel,
+			nil,
+		)
+
+		exitCode := INTERNAL_ERROR
+		incidentID := monitor.CapturePanic(func() {
+			exitCode = RunWorker()
+		})
```

Comment:
I think I prefer capturing the panic inside `RunWorker` as before, since all of the tests use `RunWorker` as their entry point - thus, we'd lose the panic handling in tests if we do it here. The main method is very slim and well tested, so I do not think it is possible for a panic to occur there.